packages/patterns/README.md

```diff
@@ -0,0 +1,27 @@
+# Patterns
+
+## Installation
+
+Install the module
+
+```bash
+npm install @wordpress/patterns --save
+```
+
+_This package assumes that your code will run in an **ES2015+** environment. If you're using an environment that has limited or no support for such language features and APIs, you should include [the polyfill shipped in `@wordpress/babel-preset-default`](https://github.com/WordPress/gutenberg/tree/HEAD/packages/babel-preset-default#polyfill) in your code._
+
+## How it works
+
+This package is used internally by the Gutenberg project to manage the creation and editing of user patterns using the `wp_block` CPT in the context of the block editor. The likes of the `PatternsMenuItems` component expect to be rendered within a `BlockEditorProvider` in order to work.
```

Comment:
Is it worth raising the prominence of this via `warning` or `note` notices, or making it the first part of the readme? I'm thinking of how the [interactivity package conveys](https://github.com/WordPress/gutenberg/blob/trunk/packages/interactivity/README.md?plain=1#L1-L11) similar info.

Reply:
Have moved it to a note at the top of the readme.